net/kristall-devel/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+PortGroup           openssl 1.0
+PortGroup           qmake5  1.0
+
+set commit_sha      6b39f24484bb0796f3f383401f95904f85b74d7b
+github.setup        MasterQ32 kristall ${commit_sha}
+github.tarball_from archive
+name                kristall-devel
+version             20211120
+revision            0
+
+homepage            https://kristall.random-projects.net
+
+description         Small-Internet Browser for Gemini, Gopher, Finger & HTTP
+
+long_description    Kristall is a browser without support for css/js/wasm or \
+                    graphical websites. It can display user-styled documents \
+                    in several formats, including gemini, html, markdown, ... \
+                    provided by a server via gemini, gopher, http, finger, ...
+
+categories          net
+license             GPL-3.0
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
```

Comment:
I tend to dislike when people split this across lines; it makes it hard to find openmaintainer ports with grep (though doubtless someone will now pipe up and say I should have been using an option to the `port` command...) That said, this is not a reason not to commit this.

Reply:
😆 Not to be be one of those piping up, but there is the following for listing `openmaintainer` ports:

```
port echo maintainer:openmaintainer
```

I'm not sure if that's the command you were looking for...